front/app/containers/IdeasNewSurveyPage/messages.ts

```diff
@@ -1,9 +1,9 @@
 import { defineMessages } from 'react-intl';
 
 export default defineMessages({
-  surveyNewMetaTitle1: {
-    id: 'app.containers.IdeasNewPage.surveyNewMetaTitle1',
-    defaultMessage: 'Add survey response to project | {orgName}',
+  surveyNewMetaTitle2: {
```

Comment:
Don't we only need to update the ID string to trigger translations and not the name? I would just change the name back to `surveyNewMetaTitle`